service/templates/layout.html

```diff
@@ -53,11 +53,23 @@
           </div>
         </div>
       {% endif %}
-      <div class="column-quarter column-flip">
-        <div class="account">
-          {% if username %}
-            {{username}}
-          {% endif %}
+      <div class="column-third column-flip">
+        <div class="language">
+          {# The following form resubmits the page if an alternate (english or welsh) #}
+          {# language is selected. The language code goes in g.locale. The form has #}
+          {# no action specified in order to resubmit the current page while maintaining #}
+          {# the query string. The route is a GET so the method is get, with 'language=' #}
+          {# added to the query string (ie, not POST + http header variables) #}
```

Comment:
This comment states that the form should submit whilst maintaining the current query string. However if you go to page 2 of the results page and switch to welsh it wipes out the query string and you actually get taken back to page 1 so I think we need to fix the code to match this comment. 

On a similar note, we apparently do not want the language switcher to appear on every page - it should only appear on the title page itself.